css/styles.css

```diff
@@ -1074,7 +1074,6 @@ nav .btn {
 .sticky-top {
     white-space: nowrap;
     overflow-y: hidden;
-    overflow-x: scroll;
```

Comment:
Are these changes on this file intentional? I'm asking because we already have lots of edits on this commit:

https://github.com/apache/kafka-site/pull/92/files

And I'm just double checking to see if any of them were unintentionally reverted in this PR.

Reply:
@guozhangwang Yes, these changes are different. I am changing &modestbranding=1&controls=2& to rel=0& for all the videos. Does this make sense?